samples/tuples/destructuring_assignment_for.jakt

```diff
@@ -0,0 +1,10 @@
+/// Expect:
+/// - output: "1, Hello\n2, World\n"
+
+function main() {
+    mut list = [("Hello", 1), ("World", 2)]
+
+    for (word, idx) in list.iterator() {
```

Comment:
🤨 why is the index after the element?
Not a problem for the test of course, just seemed odd

Reply:
I've changed the order to be more accurate